src/ShareModal.scss

```diff
@@ -0,0 +1,56 @@
+@import '_variables';
+
+.ShareModal__content {
+    padding: 2rem;
+    border-bottom: 1px solid #E4E7EB;
+}
+
+.ShareModal__form {
+    display: flex;
+    grid-template-columns: 1fr min-content;    
+    column-gap: 1rem;
+}
+
+.ShareModal__form__URL__container {
+    border: 1px solid #9DA4AF;
+    border-radius: 0.2rem;
+    max-width: 50vh;
+    padding: 0.5rem;
+}
+
+.ShareModal__form__URL {
```

Comment:
From the Figma design, I think we want to use text-overflow: ellipsis to the visible URL part on the modal.

Reply:
Fixed.